clang/lib/StaticAnalyzer/Checkers/MoveChecker.cpp

```diff
@@ -12,6 +12,7 @@
 //
 //===----------------------------------------------------------------------===//
 
+#include "Move.h"
```

Comment:
Why is this include added here?

Reply:
It is not unrelated. `::clang::ento::move::isMovedFrom` has declaration in "Move.h" but since missing this header, It is treated by clang-tidy check as internal linkage.

I think we need to add this include to make the declaration and definition in the same file.